projects/step-core/src/lib/modules/dynamic-forms/components/dynamic-field/dynamic-field.component.ts

```diff
@@ -127,7 +127,19 @@ export class DynamicFieldComponent implements ControlValueAccessor, OnDestroy {
   }
 
   private synchronizeInternalValueWithFields(): void {
-    this.value = this.internalValue?.value;
+    let fallbackValue: string | number | boolean | undefined = undefined;
+
+    switch (this.fieldType) {
+      case DynamicFieldType.string:
+        fallbackValue = '';
+        break;
+
+      case DynamicFieldType.boolean:
+        fallbackValue = false;
```

Comment:
I would set this one to undefined, else booleans might be initialised with "false"

Reply:
not really, that is handled by the code below 
<img width="412" alt="image" src="https://user-images.githubusercontent.com/8954749/218470059-1879c8c7-6310-4e4c-a62f-9b13fcc8cbfd.png">